mix.lock

```diff
@@ -1,19 +1,14 @@
 %{
-  "cowboy": {:hex, :cowboy, "1.0.4", "a324a8df9f2316c833a470d918aaf73ae894278b8aa6226ce7a9bf699388f878", [:make, :rebar], [{:cowlib, "~> 1.0.0", [hex: :cowlib, optional: false]}, {:ranch, "~> 1.0", [hex: :ranch, optional: false]}]},
```

Comment:
Run mix deps.unlock --check-unused
** (Mix) Unused dependencies in mix.lock file:

  * :cowboy
  * :cowlib
  * :earmark
  * :fs
  * :ranch

Error: Process completed with exit code 1.